using_images/db_images/mongodb.adoc

```diff
@@ -456,54 +456,76 @@ of the *rs0* replication set, as the value of `*MONGODB_REPLICA_NAME*` is set to
 === Creating the Service Pod
 
 To initialize members created by xref:creating-the-deploymentconfig[the
-deployment configuration], a _service pod_ is defined in the template. This pod
-starts MongoDB with the `initiate` argument, which instructs the container
-entrypoint to behave xref:mongo-creating-headless-service[slightly differently]
+deployment configuration], the pods are started with the `initiate` argument, 
+which instructs the container entrypoint to behave xref:mongo-creating-headless-service[slightly differently]
```

Comment:
The "[container entrypoint](https://github.com/sclorg/mongodb-container/blob/master/2.4/Dockerfile#L57)" for that image just [execs](https://github.com/sclorg/mongodb-container/blob/master/2.4/root/usr/bin/container-entrypoint) the command/args, so one needs to make it [`['run-mongod', 'initiate']`](https://github.com/sclorg/mongodb-container/blob/master/2.4/examples/replica/mongodb-clustered.json#L93-L96).